src/modules/output_module.c

```diff
@@ -71,10 +67,22 @@ static struct k_delayed_work led_pat_gps_work = {
 	.work = Z_WORK_INITIALIZER(led_pat_gps_work_fn)
 };
 
-K_MSGQ_DEFINE(msgq_output, sizeof(struct output_msg_data), 10, 4);
+/* Output module message queue. */
+#define OUTPUT_QUEUE_ENTRY_COUNT	10
```

Comment:
Out of scope here, but we should probably make queue and buffer sizes kconfigurable

Reply:
Agreed.